src/main/scala/org/broadinstitute/dsde/firecloud/EntityService.scala

```diff
@@ -124,30 +102,15 @@ object EntityClient {
 
 }
 
-class EntityClient(requestContext: RequestContext, modelSchema: ModelSchema, googleServicesDAO: GoogleServicesDAO)(implicit val executionContext: ExecutionContext)
-  extends Actor with RestJsonClient with TSVFileSupport with LazyLogging {
+class EntityService(rawlsDAO: RawlsDAO, importServiceDAO: ImportServiceDAO, modelSchema: ModelSchema, googleServicesDAO: GoogleServicesDAO)(implicit val executionContext: ExecutionContext)
```

Comment:
renamed to `EntityService` from `EntityClient` for consistency.

there are a few places in this PR where i try to do some separation of Api/Service/DAO layers. there's definitely more i can do there, but I only wanted to tackle the ones that were directly in my way. will take care of it in a followup.